src/assets/css/_sass/components/_home-made-local.scss

```diff
@@ -33,6 +33,10 @@
 
     &:hover {
       color: $color-text--orange-hover;
+
+      img {
+        opacity: 0.65;
+      }
```

Comment:
@this does not have the desired highlighting effect.
3 options here
1) use svg/sprite with both a normal and hover color and use background positioning to change them out on hover
2) inline the svg using `{% include icons/right-arrow.svg %}` and change the color with css
```
&:hover {
  fill: $color-text--orange-hover
}
```
3) image tag (which you've done here) and change color with css - I don't think the above css works for it, so you'd have to do some research (I remember something about this still not working well, also requiring base64)

Reply:
Further commit changed to the 2nd option - so you can use `fill` to specify the color.